modules/widget-visibility/widget-conditions.php

```diff
@@ -199,6 +213,19 @@ public static function widget_admin_setup() {
 		wp_localize_script( 'widget-conditions', 'widget_conditions_parent_pages', $all_parents );
 	}
 
+	public static function get_pages() {
```

Comment:
What do you think about adding a docblock here, explaining a bit why we chose to move away from Core's `get_pages()`, and maybe linking to https://core.trac.wordpress.org/ticket/51469 for future reference?

Reply:
Great suggestion! Done 👍